### ISSUE TITLE
Module Request: 7thSense Delta Media Server

### ISSUE THREAD

**Issue by Andygoes** [bitfocus/companion#158]
Describe the feature
External control of Delta Media Servers

Is this platform dependent (windows, mac, ..)?
External control is sent via UDP/TCP, so should be independent. Delta GUI and Server are both Windows only

If documentation is required to implement, do you know where to find it?
http://portal.7thsensedesign.com/user-guides/resources/ExternalControlHelp.html

Usecases
Add a couple of usecases for why this should be implemented and when/why to use it.

This could be useful for playback controls, to insert media, go to marker, trigger Delta sequences (macros), change preset channel positions, enable disable layers/media/capture channels, adjust audio levels, and so on.

**Comment by PerRoine** [comment]
I have started on this module with a few of the Play head commands, if Andygoes could clarify to which commands he thinks would be useful and also test the module as soon as it's in the core ( I will notify you)

**Comment by PerRoine** [comment]
current support:

Play, Cue ,Stop ,Rewind, Record, Advance (n), Stepback (n), Goto Marker (marker)[play], Goto frame (frame) [play]

**Comment by PerRoine** [comment]
The original module is now in Core please test some of the actions there and I will add your suggestions as soon as possible

**Comment by Andygoes** [comment]
Everything tested successfully. This is awesome!

**Comment by PerRoine** [comment]
Great I will be updating the module in the next few days  :-)

**Comment by Andygoes** [comment]
Hi PerRoine,

I've looked through your 0.0.2 code and syntax-wise, it all looks good! I'm actually hoping to use this on an event coming up in NYC if this gets rolled into a new build. It'll make for a very slick interface!

I also sat in a training today and my boss suggested that another good commands to add is:

applyshowmode // go full screen, turn off graphs, stats, VU meters, turn on warping and blending, turn on all colors

No switches or options on that. If it isn't too much trouble to ask, that would be handy.

Also, I hate always asking and never giving. If there's a way for me to contribute and extend, I think I've got the hang of the syntax in the js file to extend it in the future if necessary. Is that an option to take the burden off your team?
